### DISCUSSION TITLE
Zeek MHR

### DISCUSSION THREAD

**Discussion by argwfm** [Security-Onion-Solutions/securityonion#10767] [Unsupported Versions] [answered]
Good day all,

Is there a preferred way to disable Sensoroni analyzers? Is removing the source from `/opt/so/saltstack/default/salt/sensoroni/files/analyzers` the only option?

Thanks!

**Comment by InfosecGoon** [comment]
Sorry, by "Sensoroni analyzers" you mean the ones available for enrichment in the Cases interface?

**Reply by argwfm** [reply]
Yes and no... I'm finding my v2.3 SO forward nodes are issuing DNS TXT queries for `<hash>.malware.hash.cymru.com` for every file observed. Would like to disable this functionality, but found minimal documentation other than references to _Team Cymru Malware Hash Registry_.

**Reply by InfosecGoon** [reply, marked as answer]
Looks like that's coming from the detect-MHR.zeek framework, which is enabled by default. Try copying init.sls from /opt/so/saltstack/default/pillar/zeek/ to /opt/so/saltstack/local/pillar/zeek and removing the line that says "frameworks/files/detect-MHR".

**Reply by argwfm** [reply]
@InfosecGoon, that fixed it; thank you!
